testing-suite/staking-v4/get_infos/get_transaction_info.py

```diff
@@ -0,0 +1,32 @@
+
+import requests
+from config import DEFAULT_PROXY
+from helpers import string_to_base64
+
+def get_status_of_tx(tx_hash: str) -> str:
+    response = requests.get(f"{DEFAULT_PROXY}/transaction/{tx_hash}/process-status")
+    response.raise_for_status()
+    parsed = response.json()
+
+    if "transaction not found" in response.text:
+        return "expired"
+
+    general_data = parsed.get("data")
+    status = general_data.get("status")
+    return status
+
+
+def check_if_error_is_present_in_tx(error, tx_hash) -> bool:
+    flag = False
+    error_bytes = string_to_base64(error)
+
+    response = requests.get(f"{DEFAULT_PROXY}/transaction/{tx_hash}?withResults=True")
+    response.raise_for_status()
+
+    if error_bytes.decode() in response.text:
+        flag = True
+
+    if error in response.text:
+        flag = True
+
+    return flag
```

Comment:
1. Add empty line EOF
2.Is it really mandatory to check both error bytes and error in text? if so, could you "compress" this in a single `return error in response.text or error_bytes in response.text` ?

Reply:
1. Added empty line. 
2. In this way is visible and I don't see how will this suggestion make the code cleaner

Reply:
This way is way clearer. Normally linters or static code checkers would throw errors the way it was originally and suggest you refactor it this way.